lib/v2/nwafu/all.js

```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { URL } = require('url');
```

Comment:
```suggestion
```
This class is globally available [since node v10](https://nodejs.org/api/url.html#class-url)